docs/release-notes.md

```diff
@@ -44,9 +44,9 @@ nav_order: 9
 - Drop `extensions` section _(fcos 1.5.0+, openshift 4.13.0+)_
 - Drop `LuksOption` and `RaidOption` types _(Go API for fcos 1.5.0+,
   flatcar 1.1.0+, openshift 4.14.0-experimental)_
-- Require Go 1.18+
-- test: Add a test for fcos/v1_6_exp to detect bad partition table configurations when wipe_tables is false and the partition table is not on the root disk.
-
+- Require Go 1.18+ 
+- Added warning for bad partition table configurations when a non-root partition tables wipe_tables is false and the partition number is zero in fcos/v1_6_exp.
```

Comment:
Hmm, I would prefer to keep this at one line if we can, lets try and make it concise.

Additionally lets also look at the tense of the wording, the release notes is present tense. 

And when referencing configs you can do so like _(fcos 1.6.0-exp)_